client/disputes/test/utils.test.ts

```diff
@@ -0,0 +1,172 @@
+		reason: 'fraudulent',
+		source: 'visa',
+		order_number: 1,
+		customer_name: 'Mock customer',
+		customer_email: 'mock@customer.net',
+		customer_country: 'US',
+		status: 'needs_response',
+		created: '2023-01-01 23:59:59',
+		due_by: addHoursToCurrentDate( 20 ), // Due within 24h of current time.
+		order: {
+			number: 1,
+			customer_url: 'https://shop.local',
+			url: 'http://test.local/order/1',
+		},
+	},
+	{
+		wcpay_disputes_cache_id: 9,
+		stripe_account_id: 'acct_test',
+		dispute_id: 'dp_asd654l',
+		charge_id: 'ch_mock',
+		amount: 1000,
+		currency: 'usd',
+		reason: 'product_unacceptable',
+		source: 'visa',
+		order_number: 2,
+		customer_name: 'Mock customer',
+		customer_email: 'mock@customer.net',
+		customer_country: 'US',
+		status: 'warning_needs_response',
+		created: '2023-01-01 23:59:59',
+		due_by: addHoursToCurrentDate( 48 ), // Due > 24h of current time.
+		order: {
+			number: 2,
+			customer_url: 'https://shop.local',
+			url: 'http://test.local/order/2',
+		},
+	},
+	{
+		wcpay_disputes_cache_id: 18,
+		stripe_account_id: 'acct_test',
+		dispute_id: 'dp_asd654l',
+		charge_id: 'ch_mock',
+		amount: 2000,
+		currency: 'chf',
+		reason: 'bank_cannot_process',
+		source: 'visa',
+		order_number: 3,
+		customer_name: 'Mock customer',
+		customer_email: 'mock@customer.net',
+		customer_country: 'US',
+		status: 'needs_response',
+		created: '2023-01-01 23:59:59',
+		due_by: addHoursToCurrentDate( 72 ), // Due > 24h of current time.
+		order: {
+			number: 3,
+			customer_url: 'https://shop.local',
+			url: 'http://test.local/order/3',
+		},
+	},
+];
+
+describe( 'getDisputesNoticeString', () => {
+	beforeEach( () => {
+		global.wcpaySettings = {
+			zeroDecimalCurrencies: [],
+			connect: {
+				country: 'US',
+			},
+			currentUserEmail: 'mock@example.com',
+			currencyData: {
+				US: {
+					code: 'USD',
+					symbol: '$',
+					symbolPosition: 'left',
+					thousandSeparator: ',',
+					decimalSeparator: '.',
+					precision: 2,
+				},
+			},
+		};
+	} );
+
+	it( 'returns null if passed an empty array', () => {
+		expect(
+			getDisputesNoticeString( {
+				activeDisputes: [],
+			} )
+		).toBe( null );
+	} );
+
+	it( 'returns the correct string for a single dispute', () => {
+		expect(
+			getDisputesNoticeString( {
+				activeDisputes: [ mockDisputes[ 1 ] ],
+			} )
+		).toEqual( `Respond to a dispute for $10.00` );
+	} );
+
+	it( 'returns the correct string for a single dispute due within 24h', () => {
+		expect(
+			getDisputesNoticeString( {
+				activeDisputes: [ mockDisputes[ 0 ] ],
+			} )
+		).toEqual( `Respond to a dispute for $12.34 – last day` );
+	} );
+
+	it( 'returns the correct string for multiple disputes', () => {
+		expect(
+			getDisputesNoticeString( {
+				activeDisputes: mockDisputes.slice( 0, 2 ),
+			} )
+		).toEqual( `Respond to 2 active disputes for a total of $22.34` );
+	} );
+
+	it( 'returns the correct string for multiple disputes with multiple currencies', () => {
```

Comment:
What do you think about testing the components render the right strings etc, rather than testing this utility? IMO the util is an implementation detail that future engineers should be free to iterate on and adjust without tests getting in the way.

Reply:
I've updated the tests for the consumer of this function (`getTasks()`) in `client/overview/task-list/test/tasks.js`. This removes the need to test individual util functions, which have been removed.